core/src/main/scala/sttp/tapir/Validator.scala

```diff
@@ -35,14 +38,25 @@ object Validator extends ValidatorMacros {
 
   def min[T: Numeric](value: T, exclusive: Boolean = false): Validator.Primitive[T] = Min(value, exclusive)
   def max[T: Numeric](value: T, exclusive: Boolean = false): Validator.Primitive[T] = Max(value, exclusive)
+  def positive[T: Numeric]: Validator.Primitive[T] = Min(implicitly[Numeric[T]].zero, exclusive = false)
```

Comment:
is `0` positive? :) I'm not sure, see eg -> https://math.stackexchange.com/questions/26705/is-zero-positive-or-negative

Reply:
Good point! So, yes from a math point of view `0` is neither positive nor negative. But actually, in software development, one of the most common checks is `x >= 0`. I've often seen `0` treated as positive. 

But, maybe with can create `positive` and `zero` (probably using `inRange` since we don't have an `Eq` validator)

Reply:
Ok I've removed the `0` from the positive check. 
I've created the `zero` validator. It seems pretty useless since it is checking a fixed value but actually, you can combine it with `positive` to "trait" `0` as positive.

```scala
Validator.positive[Int].or(Validator.zero[Int])
```

I wonder if we should add `positiveOrZero` directly. What do you think? I think can be useful( and straightforward to create/mantain) 
```scala
def positiveOrZero[T: Numeric]: Validator[T] = Validator.positive[Int].or(Validator.zero[Int])
```

Reply:
I've added `positiveOrZero` as well, If you don't like it I can remove it, otherwise, we can merge 👍